lib/appsignal/config.rb

```diff
@@ -81,7 +81,9 @@ def self.determine_root_path
       :filter_metadata => [],
       :filter_parameters => [],
       :filter_session_data => [],
-      :ignore_actions => [],
+      :ignore_actions => [
+        "Rails::HealthController#show"
+      ],
```

Comment:
I think it would be better to add these to the Rails loader defaults, like the other Rails specific config. It will be part of the config of apps using both Rails and another framework.

https://github.com/appsignal/appsignal-ruby/blob/6201f5dbe662e22e682a8a324b7e996e979ef09f/lib/appsignal/integrations/railtie.rb#L43